scripts/TEST.vcf

```diff
@@ -0,0 +1,500 @@
+##fileformat=VCFv4.2
+##FILTER=<ID=PASS,Description="All filters passed">
+##FILTER=<ID=Consensus,Description="Called by two or more callers">
+##FILTER=<ID=Hotspot,Description="In mutational hotspot, as defined by ????">
+##INFO=<ID=MQ,Number=1,Type=Integer,Description="RMS mapping quality (normal sample)">
+##INFO=<ID=MQ0,Number=1,Type=Integer,Description="Number of MAPQ=0 reads (normal sample)">
+##INFO=<ID=CAL,Number=.,Type=String,Description="List of callers making this call">
+##FORMAT=<ID=AGT,Number=.,Type=String,Description="Consensus or majority genotype">
+##FORMAT=<ID=GTC,Number=.,Type=String,Description="Genotypes for Strelka2|Mutect2|VarDict|Lancet">
+##FORMAT=<ID=GT_STATUS,Number=.,Type=String,Description="Degree of unanimity of genotype">
```

Comment:
Should we provide the options in the description: 'unanimous', 'majority', 'deadlock'? Or is this a well defined concept that I am not familiar with.

Reply:
I agree on this. It's all made up, so we should add those possible values in the desc

Reply:
sounds good!